src/core/selinux-access.c

```diff
@@ -44,22 +47,19 @@ static int audit_callback(
                 size_t msgbufsize) {
 
         const struct audit_info *audit = auditdata;
-        uid_t uid = 0, login_uid = 0;
+        uid_t uid = 0;
         gid_t gid = 0;
-        char login_uid_buf[DECIMAL_STR_MAX(uid_t) + 1] = "n/a";
         char uid_buf[DECIMAL_STR_MAX(uid_t) + 1] = "n/a";
         char gid_buf[DECIMAL_STR_MAX(gid_t) + 1] = "n/a";
 
-        if (sd_bus_creds_get_audit_login_uid(audit->creds, &login_uid) >= 0)
-                xsprintf(login_uid_buf, UID_FMT, login_uid);
         if (sd_bus_creds_get_euid(audit->creds, &uid) >= 0)
                 xsprintf(uid_buf, UID_FMT, uid);
         if (sd_bus_creds_get_egid(audit->creds, &gid) >= 0)
                 xsprintf(gid_buf, GID_FMT, gid);
 
         (void) snprintf(msgbuf, msgbufsize,
-                        "auid=%s uid=%s gid=%s%s%s%s%s%s%s%s%s%s",
-                        login_uid_buf, uid_buf, gid_buf,
+                        "uid=%s gid=%s%s%s%s%s%s%s%s%s%s",
+                        uid_buf, gid_buf,
```

Comment:
why drop this?

Reply:
ah, get it now. explanation is in your PR text.

can you split this in two patches please. one that removes this, and the other that does the retrieval from dbus